stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2455,8 +2455,15 @@ internal struct _${Self}StorageHeader {
 % end
 }
 
-internal class _RawNative${Self}Buffer : 
-  ManagedBuffer<_${Self}StorageHeader, UInt8> {
+/// An instance of this class has all `${Self}` data tail-allocated.
+/// Enough bytes are allocated to hold the bitmap for marking valid entries,
+/// keys, and values. The data layout starts with the bitmap, followed by the
```

Comment:
Is all this information about layout an implementation detail, or is it exposed for direct use by other internal components?  If the former, it shouldn't be in a doc comment.